rpc/net.go

```diff
@@ -190,7 +190,7 @@ func (s *NetService) takeOffer(makerPeerID peer.ID, offerID types.Hash, provides
 
 	swapState, err := s.xmrtaker.InitiateProtocol(makerPeerID, providesAmount, offer)
 	if err != nil {
-		return fmt.Errorf("failed to initiate protocol: %w", err)
```

Comment:
At least with the error samples I was looking at (see `TestBadMakeTakeValues` in the `daemon` package), this was making the error message longer/harder to read without adding helpful context. This particular error is passed to the RPC client who was issuing a TakeOffer request, so it is a more user facing error than the ones we log.